WORKSPACE

```diff
@@ -1,3 +1,4 @@
+workspace(name = "cxx")
```

Comment:
Is this workspace name expected to be globally unique in the downstream repo? `cxx` is sufficiently unique as the name of a Rust crate, but not necessarily as an identifier in a multi language codebase. Would it be idiomatic to use e.g. `cxx.rs`?

Reply:
I like `cxx.rs`, great idea! PTAL :)